cmd/gnocco/server.go

```diff
@@ -91,10 +101,13 @@ func (s *server) start(ds *dns.Server) {
 }
 
 func (s *server) shutDown() {
+	logger := s.cf.logger
 	logger.Info().Print("Shutdown called.")
 }
 
 func (s *server) startCacheDumping() {
+	mainconfig := s.cf
```

Comment:
why this line? can't you sue s.cf.Whatever instead of mainconfig.Whatever?

Reply:
very true. it was to reduce the number of changes :-/

Reply:
otherwise  10x lines would be affected

Reply:
I can clean that up on a follow up commit after the logic of the change is reviewed